src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -154,20 +156,22 @@ const ChatItemMyTextV2 = ({
 
   return (
     <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
-      <View style={handleTextContainerStyle()}>
-        {!isContinuous && (
-          <View
-            style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
-            <Text style={styles.userText}>{username}</Text>
-            <View style={styles.dot} />
-            <Text style={styles.timeText}>{time}</Text>
-          </View>
-        )}
-        {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
-        {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
+      <ChatContextMenuView contextMenuType="MyChatContextMenu" message={message}>
+        <View style={handleTextContainerStyle()}>
+          {!isContinuous && (
+            <View
+              style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
+              <Text style={styles.userText}>{username}</Text>
+              <View style={styles.dot} />
+              <Text style={styles.timeText}>{time}</Text>
+            </View>
+          )}
+          {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
+          {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
 
-        {renderIcon()}
-      </View>
+          {renderIcon()}
+        </View>
+      </ChatContextMenuView>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new feature of long press copy is implemented here by wrapping the chat message inside `ChatContextMenuView`. This seems to be a good approach as it encapsulates the context menu logic inside its own component, improving modularity and maintainability. However, I would suggest adding error handling within the `ChatContextMenuView` component to ensure that any errors during the copying process are caught and handled appropriately, preventing the application from crashing.
```diff
// Inside ChatContextMenuView component
try {
  // Copying logic
} catch (error) {
  // Handle or throw error
}
```
This will help in maintaining the stability of the application even when unexpected issues occur.


<!-- This is an auto-generated comment by OSS CodeRabbit -->